packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -791,20 +805,32 @@ class C4DTableOfContents extends MediaQueryMixin(
             : ``}
           <div
             class="${c4dPrefix}-ce--table-of-contents__items-container"
+            part="container"
             style="position: sticky; top: ${stickyOffset &&
             this.layout !== TOC_TYPES.HORIZONTAL
               ? `${stickyOffset}px`
               : 0}">
-            <div class="${prefix}--tableofcontents-container">
+            <div
+              class="${prefix}--tableofcontents-container"
+              part="item-container">
               <div
                 class="${prefix}--tableofcontents"
+                part="content"
```

Comment:
`part="content"` is being reused a lot here. It looks to me like each place it's being used, it's a unique element that calls for a unique part name. Let's give a unique name to each.

Reply:
Please check again.